vendor/bat-native-ledger/src/bat/ledger/internal/bat_contribution.cc

```diff
@@ -630,14 +630,20 @@ void BatContribution::ReconcilePayloadCallback(
     const std::map<std::string, std::string>& headers) {
   ledger_->LogResponse(__func__, response_status_code, response, headers);
 
+  const auto reconcile = ledger_->GetReconcileById(viewing_id);
+
   if (response_status_code != 200) {
-    AddRetry(ledger::ContributionRetry::STEP_PAYLOAD,
-             viewing_id);
+    if (response_status_code == 416) {
```

Comment:
See previous comments concerning HTTP status codes

Reply:
let's do it in one sweep https://github.com/brave/brave-browser/issues/4429